securedrop/source_templates/session_timeout.html

```diff
@@ -3,6 +3,6 @@
     <img src="{{ url_for('static', filename='i/hand_with_fingerprint.png') }}">
   </div>
   <div class="message"><strong>{{ gettext('Important!') }}</strong><br>
-  <p>{{ gettext('Your session timed out due to inactivity. Please login again if you want to continue using SecureDrop, or select "New Identity" from the onion button in the Tor browser\'s toolbar to clear all history of your SecureDrop usage from this device. If you are not using Tor Browser, restart your browser.') }}</p>
+    <p>{{ gettext('You were logged out due to inactivity. Click the <img src={icon} alt="broom icon" width="16" height="16">&nbsp;<strong>New Identity</strong> button in your Tor browser\'s toolbar. This will clear your Tor browser activity data on this device.').format(icon=url_for('static', filename='i/torbroom-black.png'))  }}</p>
```

Comment:
"button in your Tor browser's toolbar" -> "button in your Tor browser's toolbar before moving on" per language in https://github.com/freedomofpress/securedrop-ux/issues/94#issuecomment-589456323